app/jsonimplicits/implicits.scala

```diff
@@ -103,26 +97,22 @@ object model {
     }
   }
 
-  implicit def referenceReads[T](implicit idLookup:ArnLookup[T]): Reads[Reference[T]] = new Reads[Reference[T]] {
-    override def reads(json: JsValue): JsResult[Reference[T]] = JsSuccess(Reference[T](json.as[String]))
-  }
+//  implicit def referenceReads[T](implicit idLookup:ArnLookup[T]): Reads[Reference[T]] = (json: JsValue) => {
+//    JsSuccess(Reference[T](json.as[String], Map.empty, prism))
+//  }
```

Comment:
If not used then let's delete it. Guessing if it compiles then it is no longer used 😄

Reply:
🛠️